e2e/app.e2e-spec.ts

```diff
@@ -14,7 +14,7 @@ describe('kimchistan-front-end App', () => {
     browser.wait(function () {
       return element(by.id('dropdown_Bibimbap')).isPresent();
     }, 5000);
-    element(by.id('dropdown_Bibimbap')).click();
+    element(by.id(`dropdown_Bibimbap`)).click();
```

Comment:
Why did you change to back-ticks?

Reply:
Probably some sneaky Islandic keyboard ;)

Reply:
This COULD be a problem though...

Reply:
Yes, I will change it today, as well as acting on the rest of your comments. :)